docs/handlers.md

```diff
@@ -0,0 +1,1451 @@
+| Format        | Type                                 | Fully supported?    |
+| :------------ | :----------------------------------- | :-----------------: |
+| [`AR`](#ar) | ARCHIVE | :octicons-check-16: |
+| [`ARC`](#arc) | ARCHIVE | :octicons-check-16: |
+| [`ARJ`](#arj) | ARCHIVE | :octicons-check-16: |
+| [`BDL`](#bdl) | ARCHIVE | :octicons-check-16: |
+| [`BNEG`](#bneg) | ARCHIVE | :octicons-check-16: |
+| [`BZIP2`](#bzip2) | COMPRESSION | :octicons-check-16: |
+| [`CAB`](#cab) | ARCHIVE | :octicons-check-16: |
+| [`CHK`](#chk) | ARCHIVE | :octicons-check-16: |
+| [`COMPRESS`](#compress) | COMPRESSION | :octicons-check-16: |
+| [`CPIO_BINARY`](#cpio_binary) | ARCHIVE | :octicons-check-16: |
+| [`CPIO_PORTABLE_ASCII`](#cpio_portable_ascii) | ARCHIVE | :octicons-check-16: |
+| [`CPIO_PORTABLE_ASCII_CRC`](#cpio_portable_ascii_crc) | ARCHIVE | :octicons-check-16: |
+| [`CPIO_PORTABLE_OLD_ASCII`](#cpio_portable_old_ascii) | ARCHIVE | :octicons-check-16: |
+| [`CRAMFS`](#cramfs) | FILESYSTEM | :octicons-check-16: |
+| [`DMG`](#dmg) | ARCHIVE | :octicons-check-16: |
+| [`ECC`](#ecc) | ARCHIVE | :octicons-check-16: |
+| [`ELF32`](#elf32) | EXECUTABLE | :octicons-check-16: |
+| [`ELF64`](#elf64) | EXECUTABLE | :octicons-check-16: |
+| [`ENCRPTED_IMG`](#encrpted_img) | ARCHIVE | :octicons-check-16: |
+| [`ENGENIUS`](#engenius) | ARCHIVE | :octicons-alert-fill-12: |
+| [`EROFS`](#erofs) | FILESYSTEM | :octicons-check-16: |
+| [`EXTFS`](#extfs) | FILESYSTEM | :octicons-check-16: |
+| [`FAT`](#fat) | FILESYSTEM | :octicons-check-16: |
+| [`GZIP`](#gzip) | COMPRESSION | :octicons-check-16: |
+| [`HDR1`](#hdr1) | ARCHIVE | :octicons-check-16: |
+| [`HDR2`](#hdr2) | ARCHIVE | :octicons-check-16: |
+| [`INSTAR_HD`](#instar_hd) | ARCHIVE | :octicons-check-16: |
+| [`IPKG`](#ipkg) | ARCHIVE | :octicons-check-16: |
+| [`ISO9660`](#iso9660) | FILESYSTEM | :octicons-check-16: |
+| [`JFFS2_NEW`](#jffs2_new) | FILESYSTEM | :octicons-check-16: |
+| [`JFFS2_OLD`](#jffs2_old) | FILESYSTEM | :octicons-check-16: |
+| [`LZ4`](#lz4) | COMPRESSION | :octicons-check-16: |
+| [`LZ4`](#lz4) | COMPRESSION | :octicons-check-16: |
+| [`LZ4`](#lz4) | COMPRESSION | :octicons-check-16: |
+| [`LZH`](#lzh) | COMPRESSION | :octicons-check-16: |
+| [`LZIP`](#lzip) | COMPRESSION | :octicons-check-16: |
+| [`LZMA`](#lzma) | COMPRESSION | :octicons-check-16: |
+| [`LZO`](#lzo) | COMPRESSION | :octicons-check-16: |
+| [`NTFS`](#ntfs) | FILESYSTEM | :octicons-check-16: |
+| [`QNAP_NAS`](#qnap_nas) | ARCHIVE | :octicons-check-16: |
+| [`RAR`](#rar) | ARCHIVE | :octicons-alert-fill-12: |
+| [`ROMFS`](#romfs) | FILESYSTEM | :octicons-check-16: |
+| [`SEVENZIP`](#sevenzip) | ARCHIVE | :octicons-check-16: |
+| [`SHRS`](#shrs) | ARCHIVE | :octicons-check-16: |
+| [`SPARSE`](#sparse) | FILESYSTEM | :octicons-check-16: |
```

Comment:
We should probably review and adjust the handler names to be more informative.

This one should be `android_sparse_image` or something along the line

Reply:
Now it's "Android Sparse Image".